hibernate6/src/test/java/com/fasterxml/jackson/datatype/hibernate6/BaseTest.java

```diff
@@ -0,0 +1,62 @@
+package com.fasterxml.jackson.datatype.hibernate6;
+
+import java.util.Arrays;
+
+import org.apache.log4j.Logger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public abstract class BaseTest extends junit.framework.TestCase
+{
+    protected BaseTest() {
+    	try {
+    		System.out.println(Hibernate6Version.getHibernateVersion());
+    		System.out.println(Hibernate6Version.isHibernate6_Plus());
+			Logger.getLogger(this.getClass()).info("Testing using hibernate " + Hibernate6Version.getHibernateVersion() +
+					", is 6+: " + Hibernate6Version.isHibernate6_Plus());
+		} catch (Exception e) {
+			// Should not happen
+			throw new RuntimeException(e);
+		}
+    }
+
+    protected ObjectMapper mapperWithModule(boolean forceLazyLoading)
+    {
+        return new ObjectMapper().registerModule(hibernateModule(forceLazyLoading, false));
```

Comment:
Let's convert to using builder:

    JsonMapper.builder().addModule(...).build();

in case there will be Jackson 3.x version of module